packages/plugin-core/src/commands/ReloadIndex.ts

```diff
@@ -241,7 +241,9 @@ export class ReloadIndexCommand extends BasicCommand<
         errors.forEach((error) => {
           if (DuplicateNoteError.isDuplicateNoteError(error) && error.code) {
             VSCodeUtils.showMessage(MessageSeverity.WARN, error.message, {});
-            AnalyticsUtils.track(WorkspaceEvents.DuplicateNoteFound);
+            AnalyticsUtils.track(WorkspaceEvents.DuplicateNoteFound, {
+              source: "ReloadIndex",
```

Comment:
for commands, can we just use `this.key`?

Reply:
We already use `DuplicateNoteFound` event when we find them during the initialization, so I think it makes sense to use the same event here.

Reply:
> for commands, can we just use this.key?

sure.

> We already use DuplicateNoteFound event when we find them during the initialization, so I think it makes sense to use the same event here.

Are you replying to @kevinslin? Otherwise I'm a bit confused what you are saying. This is the one that's being called during initialization.

Reply:
I replied without reading the code, please disregard. Sorry :sweat_smile: